src/components/Book.js

```diff
@@ -9,19 +9,38 @@ function Book({
 }) {
   const dispatch = useDispatch();
   return (
-    <li>
+    <div className="card-container">
       <div className="book">
+        <p className="cate">{category}</p>
         <h3>{title}</h3>
-        <p>{author}</p>
-        <p>{category}</p>
-        <Button
-          type="button"
-          onClick={() => dispatch(removeBook(id))}
-          btnValue="Delete"
-          btnName="remove"
-        />
+        <p className="author">{author}</p>
+        <div className="buttons">
+          <Button btnName="comments-btn" btnValue="comments" />
+          <div className="separate" />
+          <Button
+            type="button"
+            onClick={() => dispatch(removeBook(id))}
+            btnValue="Remove"
+            btnName="remove"
+          />
+          <div className="separate" />
+          <Button btnName="edit-btn" btnValue="Edit" />
+        </div>
       </div>
-    </li>
+      <div className="progress-sec">
+        <div className="oval" />
+        <div className="percentages">
+          <h2>66%</h2>
+          <p>Completed</p>
+        </div>
+        <div className="line" />
+      </div>
+      <div className="update-container">
+        <h5>CURRENT CHAPTER</h5>
+        <p>Chapter 17</p>
+        <Button btnName="update-btn" btnValue="UPDATE PROGRESS" />
+      </div>
+    </div>
```

Comment:
- [ ] To better achieve proper alignment of these three sections, it would be nice if you could place them in separate divs and use flex with space-between alignment so the percentage section stays aligned no matter the title of a book. Currently, the percentage section is not aligned and is a result of the length of a book title. But if the three sections are placed in separate divs with their respective width, you'll be able to achieve the design. Kindly take a look at the screenshot as I marked the sections and compare the two. 👍🏽